src/monitoring/aws/rds/metrics.ts

```diff
@@ -0,0 +1,100 @@
+import * as cloudwatch from '@aws-cdk/aws-cloudwatch';
+import * as cdk from '@aws-cdk/core';
+
+/**
+ * Metrics for RDS Aurora.
+ */
+export class RdsAuroraMetricFactory {
+  metricDmlThroughput(clusterIdentifier: string) {
```

Comment:
Seems like `clusterIdentifier` is common across all the metrics. Do you think we should add it as a class member in constructor? That would reduce the churn on consumers to pass the identifier for every metric. Better way would be to accept the `ICluster` interface if there is one for better expandability in future. Thoughts?

Reply:
Hello! I can try to explain. I wanted these metric factories to be basically just a group of related low-level methods, rather than resource-specific features. The factories are helping their users to avoid remembering the exact names and dimensions needed. 

Another reason is that some of the metrics (e.g. errors in Dynamo DB) can have additional dimensions, e.g. Operation. Having all dimension passed from the outside, we allow for a flexibility to provide different metrics with different dimensions.